media-sound/picard/picard-1.4.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+DISTUTILS_SINGLE_IMPL=1
+DISABLE_AUTOFORMATTING=true
+inherit eutils distutils-r1 readme.gentoo-r1
+
+DESCRIPTION="A cross-platform music tagger"
+HOMEPAGE="https://picard.musicbrainz.org"
+SRC_URI="https://github.com/metabrainz/${PN}/archive/release-${PV}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="+acoustid +cdda nls"
+
+DEPEND="dev-python/PyQt4[X,${PYTHON_USEDEP}]
+	dev-qt/qtgui:4[accessibility]
+	media-libs/mutagen
+	acoustid? ( >=media-libs/chromaprint-1.0[tools] )
+	cdda? ( dev-python/python-discid )"
+RDEPEND="${DEPEND}"
+
+RESTRICT="test" # doesn't work with ebuilds
+S=${WORKDIR}/${PN}-release-${PV}
+DOCS="AUTHORS.txt NEWS.txt"
+
+DOC_CONTENTS="If you are upgrading Picard and it does not start,
+try removing Picard's settings:
+	rm ~/.config/MusicBrainz/Picard.conf"
+
+src_compile() {
+	distutils-r1_src_compile $(use nls || echo "--disable-locales")
```

Comment:
I had a look into this and mydistutilsargs only sets global arguments to setup.py, not arguments specific to each phase (for example, build and install). It appears that local arguments for build and install phases have to be passed in separately using the distutils-r1_python_compile [additional-args...] and distutils-r1_python_install [additional-args...] approaches.

Unfortunately "use_enable" will not work in this situation as passing in "--enable-locales" is a fatal condition to setup.py. Additionally, I tried using "usex" with a blank argument and also encountered a fatal condition as this results in an extra space in the arguments list which again is a fatal condition to setup.py.